src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoGetMonthName.cs

```diff
@@ -2,145 +2,44 @@
-        [Fact]
-        public void PosTest1()
-        {
-            DateTimeFormatInfo info = CultureInfo.InvariantCulture.DateTimeFormat;
-            string[] expected = new string[] {
-                "",
-                "January",
-                "February",
-                "March",
-                "April",
-                "May",
-                "June",
-                "July",
-                "August",
-                "September",
-                "October",
-                "November",
-                "December",
-                "",
-            };
-
-            VerificationHelper(info, expected);
-        }
-
-        // PosTest2: Call GetMonthName on en-us culture DateTimeFormatInfo instance
-        [Fact]
-        public void PosTest2()
+        public static IEnumerable<object[]> GetMonthName_TestData()
         {
-            DateTimeFormatInfo info = new CultureInfo("en-us").DateTimeFormat;
-            string[] expected = new string[] {
-                "",
-                "January",
-                "February",
-                "March",
-                "April",
-                "May",
-                "June",
-                "July",
-                "August",
-                "September",
-                "October",
-                "November",
-                "December",
-                "",
-            };
-
-            VerificationHelper(info, expected);
-        }
-
-        // PosTest3: Call GetMonthName on fr-FR culture DateTimeFormatInfo instance
-        // When dotnet/corefx#2103 is addressed, we should remove the conditional fact attribute and select the right data for fr-FR
-        [ConditionalFact("IsNotUbuntu1510")]
-        public void PosTest3()
-        {
-            DateTimeFormatInfo info = new CultureInfo("fr-FR").DateTimeFormat;
-            string[] expected = new string[] {
-                "",
-                "janvier",
-                "f\u00E9vrier",
-                "mars",
-                "avril",
-                "mai",
-                "juin",
-                "juillet",
-                "ao\u00FBt",
-                "septembre",
-                "octobre",
-                "novembre",
-                "d\u00E9cembre",
-                "",
-            };
-
-            VerificationHelper(info, expected);
-        }
+            yield return new object[] { DateTimeFormatInfo.InvariantInfo, new string[] { "", "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December", "" } };
+            yield return new object[] { new DateTimeFormatInfo(), new string[] { "", "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December", "" } };
+            yield return new object[] { new CultureInfo("en-US").DateTimeFormat, new string[] { "", "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December", "" } };
```

Comment:
All three of these use the same data.  Can we create the array just once, and pass it with all three?  This applies in general.